eng/Analyzers.props

```diff
@@ -12,7 +12,7 @@
     <PackageReference Include="Microsoft.CodeAnalysis.BannedApiAnalyzers" PrivateAssets="all" />
 
     <AdditionalFiles Include="$(RepositoryEngineeringDir)stylecop.json" Link="stylecop.json" />
-    <PackageReference Include="StyleCop.Analyzers" PrivateAsset="all" />
+    <PackageReference Include="StyleCop.Analyzers" PrivateAssets="all" />
```

Comment:
This typo was causing `StyleCopAnalyzers` packages to be added as deps for packages where we don't have a `.nuspec`.